src/settings.html

```diff
@@ -14,6 +14,10 @@ <h3 class="hr-heading">Login</h3>
 				<hr />
 				<input type="email" id="username" placeholder="Email">
 				<br>
+      	<div id="emailWarning" style="display:none; color:red; margin-left: 10px">
```

Comment:
Please do the proper indentation.

Reply:
@samagragupta I've changed the indentation, but somehow it looks different on browser. You can see the screenshot from the codebase, it looks fine. 
![screenshot 2018-12-08 at 7 58 53 pm](https://user-images.githubusercontent.com/31539812/49687091-f1906780-fb23-11e8-8f88-1b685e700e06.png)